src/styles.css

```diff
@@ -60,15 +60,32 @@ body {
   color: #91caff;
 }
 
+
+/* BEGIN about HTML styling */
+
+/* Header elements which are the first child of the about content container shouldn't have the default top margin */
+.about-content h1:first-child, h2:first-child, h3:first-child, h4:first-child, h5:first-child, h6:first-child {
```

Comment:
We can also consider maybe just affecting the first child of about content instead of just the headers as first child?

Reply:
done